bukkit/src/main/java/dev/jsinco/brewery/bukkit/TheBrewingProject.java

```diff
@@ -123,9 +123,11 @@ public void initialize() {
 
     @Override
     public void onLoad() {
-        instance = this;
+        saveResources();
+        initialize();
```

Comment:
If something fails on load, the plugin loading will continue on to the enable stage. I guess we should add some check for that as well